core/new-gui/src/app/workspace/component/property-editor/operator-property-edit-frame/operator-property-edit-frame.component.scss

```diff
@@ -1,3 +1,8 @@
+::ng-deep .highlighted .mat-form-field-flex {
+  outline: 3px solid green;
```

Comment:
the `green` seems to be hardcoded. see if @jessieeeovo and @sadeemsaleh have the need to have the color configurable?